src/core/BufferAttribute.js

```diff
@@ -42,6 +43,13 @@ class BufferAttribute {
 
 	}
 
+	get updateRange() {
```

Comment:
@gkjohnson Please add a `// @deprecated, r159` comment here.

Reply:
Feel free to make a PR. This one is merged

Reply:
Note that since this only defines a getter, setting `updateRange` is in fact broken in r160.

This broke our project and we were able to work around it easily, but I think it would be nice if the release notes for r160 were updated to mention the sudden breaking change for setting the value. (Or you could introduce a setter with a deprecation message.)

Reply:
Breaking changes are separately listed in the migration guide along with the migration tasks: https://github.com/mrdoob/three.js/wiki/Migration-Guide#r158--r159

Reply:
Aha! Unfortunately, that `updateRange` does not appear on [this page](https://github.com/mrdoob/three.js/releases) when I search for it. I had to spend a few minutes trying to figure out why setting `updateRange` wasn't working, because nothing in the release notes actually mentioned the removal of the setter. While I understand the appeal of a condensed migration reference, it feels like the release notes themselves are sort of pretending that certain changes don't exist — when in fact those are the most impactful changes for existing codebases.

Would it be reasonable to inline a copy of the migration changes into the release notes, or to include the breaking changes directly as bullets in the list of changes? I think this would make the release notes a lot more intuitive, since other projects include breaking changes in the release notes themselves.